src/pages/singleLineMap/index.tsx

```diff
@@ -40,7 +43,7 @@ const position: Point = {
 const SingleLineMapPage = () => {
   const { search, setSearch } = useContext(SearchContext)
   const { operatorId, lineNumber, timestamp, routes, routeKey } = search
-
+  const [isExpanded, setIsExpanded] = useState<Boolean>(false)
```

Comment:
```suggestion
  const [isExpanded, setIsExpanded] = useState<Boolean>(false)
  const toggleExpanded = useCallback(()=>setIsExpanded(expanded => !expanded), [])
```
using `toggleExpanded` will let you have the same function reference